conda/conda-build/meta.yaml

```diff
@@ -53,6 +61,12 @@
 {% set debug_tag='' %}
 {% endif %}
 
+{% if ucx_configured_bool %}
+{% set ucx_tag='_ucx' %}
```

Comment:
indentation?

Reply:
Sorry, Didn't get this. It follows the other settings.

Reply:
ah it does :(